ui/src/components/logs/ContainerLogsView.js

```diff
@@ -121,45 +126,57 @@ export const ContainerLogsView = ({
 
   const [logUrl, setLogUrl] = useState("");
   const [stackdriverUrl, setStackdriverUrl] = useState("");
+  const [imagebuilderStackdriverUrl, setImagebuilderStackdriverUrl] = useState("");
 
   useEffect(
     () => {
-      if (params.component_type !== "" && projectLoaded) {
-        const activeContainers = containers.filter(
-          (container) => container.component_type === params.component_type
-        );
-
-        if (activeContainers && activeContainers.length > 0) {
-          const containerQuery = {
-            ...params,
-            cluster: activeContainers[0].cluster,
-            namespace: activeContainers[0].namespace,
-            timestamps: true,
-            project_name: project.name,
-            model_id: model.id,
-            model_name: model.name,
-            version_id: versionId,
-            revision_id: revisionId ? revisionId : "",
-            prediction_job_id: jobId ? jobId : "",
-          };
-          const logParams = new URLSearchParams(containerQuery).toString();
-          const newLogUrl = config.MERLIN_API + "/logs?" + logParams;
-          if (newLogUrl !== logUrl) {
-            setLogUrl(newLogUrl);
-          }
+      if (projectLoaded) {
+        // set image builder url
+        let stackdriverQuery = {
+          job_name: project.name + "-" + model.name + "-" + versionId
+        };
+        setImagebuilderStackdriverUrl(createStackdriverUrl(stackdriverQuery, "image_builder"));
```

Comment:
this will always be created, even when there are no active components. 
The query combined with the image builder information by config, will allow this to work when there are no active components